doc/releases/changelog-0.28.0.md

```diff
@@ -734,30 +586,28 @@
   ```python
   def statistics(self, circuit: QuantumTape, shot_range=None, bin_size=None):
   ```
-
   [(#3421)](https://github.com/PennyLaneAI/pennylane/pull/3421)
 
-* The `MeasurementProcess.return_type` argument has been removed from the `__init__` method. Now
-  it is a property of the class.
+* The `MeasurementProcess.return_type` argument has been removed from the `__init__` method. Now it is a property of the class.
   [(#3434)](https://github.com/PennyLaneAI/pennylane/pull/3434)
 
 * The `MeasurementProcess` class is now an abstract class.
   [(#3434)](https://github.com/PennyLaneAI/pennylane/pull/3434)
```

Comment:
@Jaybsoni How about this to address the comment above?
```suggestion
* The `MeasurementProcess` class is now an abstract class and `return_type` is now a property of the class.
  [(#3434)](https://github.com/PennyLaneAI/pennylane/pull/3434)
```

Reply:
Perfect 💯